base/nullable.jl

```diff
@@ -3,7 +3,7 @@
 immutable NullException <: Exception
 end
 
-Nullable{T}(value::T, isnull::Bool=false) = Nullable{T}(value, isnull)
+Nullable{T}(value::T, hasvalue::Bool=true) = Nullable{T}(value, hasvalue)
```

Comment:
> is the second argument meant to be used?

@tkelman Indeed the two-argument form is mostly useful for tests, where using `===` requires controlling the contents of `value` (cf. https://github.com/JuliaLang/julia/issues/16923). Maybe it can also help for performance, instead of writing `ifelse(hasvalue, Nullable(value), Nullable())`, I would have to check. I guess @johnmyleswhite can tell.